source/data_at_rest_encryption.rst

```diff
@@ -77,10 +58,10 @@ the encryption. By default, the |opt.encryption-cipher-mode| option uses the
    --outputPath              The path to save the decrypted rollback file
    ========================  ==================================================================================
 
-.. rubric:: Important Configuration Options
+.. rubric:: Important Configuration Options When Using Key File Encryption
 
-|PSMDB| supports the ``encryptionCipherMode`` option where you choose one of the
-following cipher modes:
+With encryption key files, |PSMDB| supports the ``encryptionCipherMode`` option
+where you choose one of the following cipher modes:
 
 - |mode.cbc|
 - |mode.gcm|
```

Comment:
`encryptionCypherMode` is supported regardless where the encryption key is stored